www/src/components/graphql-embed.js

```diff
@@ -0,0 +1,49 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { useRef, useState, useCallback, useEffect } from "react"
+import PropTypes from "prop-types"
+
+export default function GraphqlEmbed({ lazy, title, url, query }) {
+  const ASPECT_RATIO = 2 / 3
```

Comment:
I'm not sure this is the best approach. In `querying-data`, the width/height are given as 600/300 (for an aspect ratio of  1 / 2). While I'm okay with changing the aspect ratio for consistency, I'd like to leave it up to the docs team (ccing @gillkyle).

Reply:
Sure I can switch the aspect ratio to 1 / 2 to keep consistency.

Reply:
Well… the problem is some pages have a width/height of 600/400 (for an aspect ratio of 2/3 like you had before). Like I said, I'd be okay with keeping it consistent at 2/3 but I just want approval from the docs team to do so.